CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ option( strict_compile
 # Compile flags
 set( common_flags "-Wall" "-Wextra" "-Wpedantic" )
 set( strict_flags "-Werror" )
-set( release_flags "-O3" )
+set( release_flags "-O3" "-Wno-unknown-warning-option" )
```

Comment:
@nathangibson14 I'm not sure this is the right way to address #106 and #116. It solves the problem, but it may just be a cop-out.